Assets/MixedRealityToolkit-Examples/MixedReality.Toolkit.Examples.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.MixedReality.Toolkit.Examples</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,MixedReality</owners>
+    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/License.txt</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Examples of the Mixed Reality Toolkit</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>Unity MixedReality</tags>
+    <dependencies>
+      <dependency id="Microsoft.MixedReality.Toolkit.SDK" version="$version$" />
```

Comment:
also depends on Microsoft.MixedReality.Toolkit (or do dependencies chain?)

Reply:
Dependencies do chain. If you'd like I can add it here for explicitness.

Reply:
No need. Thanks!

Reply:
No need!